seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/pom.xml

```diff
@@ -54,4 +54,10 @@
             <artifactId>calcite-druid</artifactId>
         </dependency>
     </dependencies>
+    <repositories>
+        <repository>
+            <id>clojars</id>
+            <url>https://repo.clojars.org/</url>
+        </repository>
+    </repositories>
```

Comment:
we don't need to add this.

Reply:
Because I can't download `clojars`. I delete it.